src/features/shared-link-settings-modal/AllowDownloadSection.js

```diff
@@ -35,23 +37,35 @@ const AllowDownloadSection = ({
         </div>
     );
 
+    const tooltipMessage = { ...messages.directDownloadBlockedByAccessPolicy };
+    const downloadWrapperClass = isDirectLinkUnavailableDueToAccessPolicy ? 'bdl-is-disabled' : null;
```

Comment:
So state should not exist on its own. So you need to have a parent class.

downloadWrapperClass = classnames('bdl-AllowDownloadSection', { bdl-is-disabled: isDirectLinkUnavailableDueToAccessPolicy});

and in CSS

.bdl-AllowDownloadSection {
    &.bdl-is-disabled { ... }
}

Reply:
currently `.bdl-is-disabled` is nested within `.shared-link-settings-modal` in css,
should we still add `bdl-AllowDownloadSection` here ?

Reply:
you can have that outside

Reply:
but inside AllowDownloadSection.jsAllowDownloadSection.scss instead